relay-event-schema/src/protocol/event.rs

```diff
@@ -480,6 +480,11 @@ pub struct Event {
     #[metastructure(omit_from_schema)] // we only document error events for now
     pub breakdowns: Annotated<Breakdowns>,
 
+    /// Information about attempts to scrape a JS source or sourcemap file from the web.
+    /// This field is populated by sentry.
+    #[metastructure(omit_from_schema)] // not part of external schema
+    pub scraping_attempts: Annotated<Array<Value>>,
```

Comment:
Should we make this a `Value` instead of `Array` instead?

Reply:
I think this is a good call. Then Relay dies not even have to be updated if the sentry-side schema ever changed to an object to include more metadata, for example. Updated now.